pocs/camera/sbigudrv.py

```diff
@@ -118,6 +122,17 @@ def __del__(self):
         _ctypes.dlclose(self._CDLL._handle)
         del self._CDLL
```

Comment:
I'm curious if you ever see an AttributeError from this line when testing. The attribute isn't initialized directly in the ctor (e.g. to None), so an exception in the ctor (`__init__`) could then generate another exception in the dtor (`__del__`).

Reply:
I don't recall seeing any.

`self._CDLL` is initialised on line 66 by calling the constructor of the `ctypes.CDLL` class, which should always have a `._handle` attribute.

If I recall correctly the destructor completes with no errors but still fails to do what it was added to do, which was to allow unloading and reloading of the SBIG driver library without having to restart the Python interpreter.